### DISCUSSION TITLE
Request additional support for GraphRAG

### DISCUSSION THREAD

**Discussion by alex13by** [open-webui/open-webui#3665] [General]
**Is your feature request related to a problem? Please describe.**
Microsoft open source GraphRAG has much stronger performance than traditional RAG, and I hope to add this technology

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**
https://github.com/microsoft/graphrag

**Comment by edisonzf2020** [comment]
+1

**Comment by flyfox666** [comment]
+1

**Comment by IuvenisSapiens** [comment]
+1

**Comment by DuyTa506** [comment]
+1

**Comment by mdlmarkham** [comment]
+1

**Comment by ddwinhzy** [comment]
+1

**Comment by silentoplayz** [comment]
Related - https://github.com/open-webui/open-webui/issues/1293
Also, +1 from me as well

**Reply by oatmealm** [reply]
GraphRAG is incredibly slow as far as I can tell. What about LightRAG, which seems to focus on performance and excels in local search?

**Comment by fishfree** [comment]
+1

**Comment by lpb1** [comment]
+1

**Comment by flefevre** [comment]
My 2 cents:

- graphrag has been released today in version 1.0.0 https://github.com/microsoft/graphrag/releases/tag/v1.0.0

- lightrag seems to be more powerfull : https://github.com/HKUDS/LightRAG as they stand in the following paper : https://arxiv.org/pdf/2410.05779
